src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,20 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Failed to login user with username '{}' "
+                    + "due to authentication error: {}", "bob", e.getMessage());
```

Comment:
we should also pass the exception to the logger to maintain it there
```suggestion
            logger.error("Failed to login user with username '{}' "
                    + "due to authentication error: {}", "bob", e.getMessage(), e);
```